lib/util/api.ts

```diff
@@ -57,3 +57,17 @@ export function combineQueryParams(
   }
   return qs.stringify(search, { arrayFormat: 'repeat' })
 }
+
+/**
+ * Drops unused params so they don't show up in URL.
+ * TODO: Remove those params from core-utils.
```

Comment:
Removing these from core-utils would be a big project, because they're still relied on by some other packages which are used in custom uis by some users. I think we should try to remove the dependency on them from OTP-RR instead.

Reply:
You are right. The stuff that comes out of `getDefaultQuery` is not very useful anyway:
```
bikeSpeed: 3.58
date: "2024-06-11"
departArrive: "NOW"
endTime: "09:00"
from: null
intermediatePlaces: Array []
maxBikeTime: 20
maxEScooterDistance: 4828
maxWalkTime: 15
mode: "WALK,TRANSIT"
numItineraries: 3
otp2: true
routingType: "ITINERARY"
startTime: "07:00"
time: "10:36"
to: null
watts: 250
wheelchair: false
```